nx584/Dockerfile

```diff
@@ -15,7 +15,8 @@ RUN which python3
 RUN /usr/bin/nx584_server --help
 
 WORKDIR /usr/app/src
-COPY config.ini .
+# config.ini to be created in run.sh
```

Comment:
`config.ini` is not "created" in `run.sh`. It is passed to `run.sh` via the `/data` folder, from the outside. Maybe change the wording of the comment to:
```
# config.ini to be passed to run.sh via the externally mapped `/data` folder
```